setup.py

```diff
@@ -4,9 +4,14 @@
 
 from setuptools import setup, Extension, find_packages
 from setuptools.command.build_py import build_py
-from nml import version_info
-NML_VERSION = version_info.get_and_write_version()
 
+try:
+    from nml import version_update
+    NML_VERSION = version_update.get_and_write_version()
+except ImportError:
```

Comment:
in normal usage, version_info would be used, not version_update, so perhaps that should go first?

Reply:
When you create a bundle you want to update the version first. When ran from a bundle it will silently fall back to version_info.

Reply:
What @glx22 says.